source/locale/en/symbols.dic

```diff
@@ -181,16 +168,115 @@ _	line	most
 ‡	double dagger	some
 ‣	triangular bullet	none
 ✗	x-shaped bullet	none
+⊕	circled plus	none
+⊖	circled minus	none
+⇄	right arrow over left arrow	none
 
-#Mathematical Operators U+2200 to U+220F
-∀	for all	none
-∁	complement	none
+#Arithmetic operators
++	plus	some
+−	minus	some
+×	times	some
+⋅	times	some
+⨯	times	some
+∕	divided by	some
+⁄	divided by	some
+÷	divide by	some
+∓	minus or plus	some
+±	Plus or Minus	some
+
+#Set operations
+∖	set minus	none
+⨯	set times	none
+⊍	set union	none
+𝒫	power set of the set	none
+𝔓	power set of the set	none
+℘	power set of the set	none
+∁	complement	of the set	none
+
+#Set relations and set constructions
+∅	empty set	none
+⊂	subset of	none
+⊄	Not a subset of	none
+⊃	superset of	none
+⊅	Not a superset of	none
+⊆	subset of or equal to	none
+⊈	neither a subset of nor equal to	none
+⊇	superset of or equal to	none
+⊉	neither a superset of nor equal to	none
+⊌	Multiset	none
+
+#Equality signs
+=	equals	some
+≃	Asymptotically equal to	none
+≄	Not asymptotically equal to	none
+≅	Approximately equal to	none
+≆	Approximately but not actually equal to	none
+≈	Almost Equal to	none
+≌	all equal to	none
+≍	Equivalent to	none
+≭	Not equivalent to	none
+≎	Geometrically equivalent to	none
+≑	Geometrically equal to	none
+≚	Equiangular to	none
+≬	Between	none
+≠	Not equal to	none
+≡	Identical to	none
+≣	strictly identical to	none
+≢	Not identical to	none
+∼	similar to	none
+≙	coresponds to	none
+≟	questioned equal to	none
+
+#comparison signs
+<	less	some
+>	greater	some
+≤	less- than or equal to	none
+≦	less- than or equal to	none
+≪	much smaller than	none
+≥	greater-than or equal to	none
+≧	greater-than or equal to	none
+≫	much bigger than	none
+≶	Less than or greater than	none
+≷	Greater than or less than	none
+≮	Not less than	none
+≯	Not greater than	none
+
+#Mathematical constants
+π	Pi	some

```

Comment:
This one cannot be added until #5194 is fixed, otherwise we would break normal reading for Greek speaking people. Please remove!

Reply:
My intention is to add greek letters to the symbols-dic after having this merged.

Reply:
Please read #5194 carefully. It is not enough to just add them to the symbols file - modifications of the Python code   which handles symbols are also required.